test/e2e/run_networking_test.go

```diff
@@ -876,7 +876,7 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 
 		inspectOut := podmanTest.InspectContainer(name)
 		Expect(inspectOut[0].NetworkSettings).To(HaveField("IPAddress", ""))
-		Expect(inspectOut[0].NetworkSettings.Networks).To(BeEmpty())
+		Expect(fmt.Sprintf("%v", inspectOut[0].NetworkSettings.Networks)).To(ContainSubstring("ns:/run/netns/" + networkNSName))
```

Comment:
shouldn't this still check for empty because ns:... should not show up in the map

I am bit confused because your system test checks for the empty map and both pass? How is this possible?